src/OpenPr0gramm/Pr0grammApiClient.cs

```diff
@@ -127,5 +148,7 @@ public interface IPr0grammApiClient : IDisposable
         CookieContainer GetCookies();
         string GetCurrentNonce();
         string GetCurrentSessionId();
+        Pr0grammMeCookie GetMeCookie();
```

Comment:
Die Idee war hier eigentlich, dass sich der User den CookieContainer (mit GetCookies) holt und wegspeichert. Warum geht das in deinem Fall nicht?

Reply:
Kannst du nicht einfach den kompletten Cookie-Container speichern/serialisieren? Oder brauchst du Zugriff auf die Daten im Me-Cookie?

Reply:
Im Prinzip geht auch das, ist mir aber unlieb. Den Zugriff auf die Daten im Me-Cookie ist auch nicht schlecht, um an den eigenen Benutzernamen zu kommen.

Reply:
Für den Nutzernamen gibt es einen eigenen Endpunkt, der auch die App verwendet:
https://pr0gramm.com/api/user/name

Kann sein, dass der noch nicht in dieser API hinterlegt ist. Müsste man dann auch machen.

Reply:
Okay, unter dem Aspekt hast du recht.

Reply:
Das geht nicht im Einklang mit #13, darum bau ich flott die Ein/Ausgabe des Objektes aus und mach das Objekt wieder internal. Zeitgleich bau ich den Endpunkt flott ein.